src/main/java/org/wso2/maven/humantask/artifact/HumanTaskMojo.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Thanks. New license config is used in https://github.com/wso2/maven-tooling-bps/pull/10/commits/c530e0fc67bd76482fdb3b4cd42bcfa54feecb02